lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/ipv-core-main-journey.yaml

```diff
@@ -490,10 +492,43 @@ CRI_HMRC_KBV_J6:
     end:
       targetState: IPV_SUCCESS_PAGE
     fail-with-no-ci:
-      targetState: PRE_KBV_TRANSITION_PAGE_J3
+      targetState: PRE_EXPERIAN_KBV_TRANSITION_PAGE
     next:
       targetState: PYI_NO_MATCH
 
+# Common pages
```

Comment:
Could these be put in the shared states at the top of this file?

Reply:
In the second PR, we will remove this, but I added it to ensure that the two pages are the same.